### PR TITLE
Create international-affairs.csl

### PR DESCRIPTION
**Pull request by POBrien333**
https://forums.zotero.org/discussion/105844/style-request-international-affairs#latest

### CODE REVIEW

**Comment by bwiernik** [comment]
@adam3smith Do you know why the CI is failing? The style file is passing the Validator

**Comment by adam3smith** [comment]
@bwiernik -- no, let me test out a couple of things, sorry for the noise this will cause (In case you're not aware, the validator does have, in the pull down, an option to switch the validation schema to a strict schema which is the one we're using for CI. This style passes that, too).

**Comment by adam3smith** [comment]
The was a square bracket in the second documentation link. I'm not sure why that would break anything, but I'm guessing some broken XML parser or regex (the link works with or without it).

File: international-affairs.csl [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+          <number variable="edition" form="ordinal"/>
+          <label variable="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
```
Comment (adam3smith) [inline international-affairs.csl:43]: @POBrien333 are you sure this shouldn't have a `<substitute>` element? See the Beyond Varieties... sample citation, which looks weird.

Comment (bwiernik) [reply international-affairs.csl:43]: If we don't have an indication otherwise, let's include our standard editor -> title substitution

Comment (POBrien333) [reply international-affairs.csl:43]: Thanks for spotting. Sure.

**Pull request merged**